frontend/express/public/javascripts/countly/vue/components/vis.js

```diff
@@ -754,9 +754,23 @@
             };
         },
         computed: {
+            hasAllEmptyValues: function() {
+                var options = _merge({}, this.baseOptions, this.mixinOptions, this.option);
```

Comment:
instead of doing the merge here again.
can't we just do 
`var options = this.mergedOptions` ?